datafusion/sqllogictest/test_files/csv_files.slt

```diff
@@ -33,7 +33,7 @@ c2 VARCHAR
 ) STORED AS CSV
 WITH HEADER ROW
 DELIMITER ','
-OPTIONS ('escape' '\"')
```

Comment:
This test was wrong, since the changed code was using `as_bytes()[0]`, but now we directly reject if there is more than one char. `\` is not an escape for `.slt` files.